src/InvestJob.jsx

```diff
@@ -73,59 +90,68 @@ export class InvestJob extends React.Component {
     this.setSessionID = this.setSessionID.bind(this);
   }
 
-  saveState(event) {
-    // Save a snapshot of this component's state to a JSON file.
+  saveState() {

```

Comment:
Ha, oh I think I get it relating to the bind comment.

Reply:
Oh good! I'll explain it for myself anyway, just to convince myself that I also understand it. The `saveState` method wants to access properties of it's parent component (`InvestJob`). And so the method needs to be bound to that component. Once bound, `saveState`'s `this` object is also `InvestJob`'s `this`, so we can call `this.setState()` from inside the method, for example.

If not bound, `this` inside of `saveState` will not refer to the parent component's `this` at all.